platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/EventIntake.java

```diff
@@ -190,6 +262,17 @@ private static boolean isNotConsensus(final EventImpl event) {
      */
     private void handleConsensus(final ConsensusRound consensusRound) {
         if (consensusRound != null) {
+
+            // If we are asynchronously prehandling transactions, we need to
+            // wait for prehandles to finish before proceeding. It is critically
+            // important that prehandle is always called prior to handleConsensusRound().
+            if (prehandlePool != null) {
+                final Instant start = time.now();
```

Comment:
It's more accurate to use `time.nanoTime()`

Reply:
converted to `time.nanoTime()`